src/i18n/yfm-table/ru.json

```diff
@@ -5,5 +5,14 @@
   "row.add.before": "Добавить строку до",
   "row.add.after": "Добавить строку после",
   "row.remove": "Удалить строку",
-  "table.remove": "Удалить таблицу"
+  "table.remove": "Удалить таблицу",
```

Comment:
"table.remove" and "table.menu.column.remove" are same
Maybe we should leave one key?